### PR TITLE
feat(condo): DOMA-10515 complete blocked employees

### PR DESCRIPTION
**Pull request by tolmachev21**
Problem Description: When an employee is blocked in the system, there is no visual indication of their status. This creates confusion when it is impossible to quickly determine in the list of employees who is an active employee and who is blocked

Goal: Eliminate confusion and simplify the management of employees in the system by clearly visualizing their status

Now everything looks like this: 
Screenshot 2024-11-06 at 20 10 40

### CODE REVIEW

**Review by toplenboren — APPROVED** [review]

**Review by sitozzz — APPROVED** [review]
Screenshoot of your change is good, but also it'll be great to provide state before your changes

File: apps/condo/domains/common/components/containers/GlobalStyle.tsx [hunk 1]
```diff
@@ -339,6 +339,10 @@ export default function GlobalStyle () {
                   cursor: pointer;
                 }
               }
+
+              .ant-table-row-blocked .ant-typography {
```
Comment (pahaz) [inline apps/condo/domains/common/components/containers/GlobalStyle.tsx:343]: It's better to define inside the table component

File: apps/condo/domains/organization/hooks/useTableColumns.tsx [hunk 2]
```diff
@@ -64,6 +66,17 @@ export const useTableColumns = (
         )
     }, [intl, organizationEmployeeSpecializations])
 
+    const renderBlockedEmployee = useCallback((name) => {
+        return (
+            <>
+                {render(name)}
+                <Typography.Paragraph style={{ color: colors.red[5] }}>
+                    ({BlockedMessage})
+                </Typography.Paragraph>
```
Comment (Alllex202) [inline apps/condo/domains/organization/hooks/useTableColumns.tsx:75]: You should use typography from our ui kit

File: apps/condo/pages/employee/index.tsx
Comment (Alllex202) [inline apps/condo/pages/employee/index.tsx:1]: same ^^

**Pull request merged**